client/components/main/layouts.jade

```diff
@@ -7,6 +7,9 @@ head
     difficult to do that cleanly with Blaze -- at least without adding extra
     packages.
   link(rel="shortcut icon" href="/wekan-favicon.png")
+  script(src="https://use.typekit.net/sye2guq.js")
```

Comment:
Should reference an internal file (same origin as wekan server) instead of an external file in order to:
- protection of private life
- security issues
- instability due to an external update of the file
- possibility to use wekan in a closed network